soc/riscv/riscv-privilege/gd32vf103/Kconfig.defconfig.gd32vf103xxxx

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2021 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
```

Comment:
nit: file name should be `Kconfig.defconfig.gd32vf103`

Reply:
I'll correct it.